include/common/Platform.h

```diff
@@ -0,0 +1,20 @@
+#ifndef PLATFORM_H
+#define PLATFORM_H
+
+/*
+ * Utilities to abstract platform-specific code
+ */
+
+#ifdef _WIN32
+#include <winsock.h>
+#else
+#include <arpa/inet.h>
+#endif
+#include <cstdint>
+
+namespace Common::Platform
```

Comment:
What do you think about in the future making this platform a class? That was it can be injectable into other classes and functions to make platform code easier to handle.